src/__testUtils__/expectJSON.js

```diff
@@ -0,0 +1,51 @@
+import { expect } from 'chai';
```

Comment:
Backported and stripped type information from this module on `main`:

https://github.com/graphql/graphql-js/blob/main/src/__testUtils__/expectJSON.ts